### DISCUSSION TITLE
What is `packageManager` field in the package.json that `yarn init` creates?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [yarnpkg/berry#3870] [Q&A] [answered]
When I run `yarn init`, yarn creates a package.json file with the following content:
```
{
  name: 'name',
  packageManager: 'yarn@3.1.0'
}
```
What Is `packageManager`? should I keep it? I can't find any documentation about this.

**Comment by 7rulnik** [comment, marked as answer]
@ahhshm this field is necessary for [corepack](https://nodejs.org/api/corepack.html)

**Reply by amirhhashemi** [reply]
Thank you @7rulnik. I wasn't familiar with corepack. One question came to my mind. I have a monorepo, should this property exist in the packages' package.json file and the root package.json or just the root package.json?

**Reply by arcanis** [reply]
Just the root

**Reply by amirhhashemi** [reply]
Ok, Thanks.
